README.md

```diff
@@ -30,4 +30,32 @@ Init db
 
 ```
 paster --plugin=ckanext-ytp-comments initdb --config={ckan.ini}
-```
\ No newline at end of file
+```
+
+## Follow / Mute comments
+
+Comment notifications (via email) are managed by opt-in, i.e. without opting in to receive comment notifications at the content item or thread level, only authors or organisation admins will receive email notifications. 
+
+This feature allows users to following or mute comments at the content item level or for a specific comment thread on the content item.
+
+When a user follows comments on a content item or content item thread they will receive email notifications when new comments or replies are posted.
+
+### Setup
+
+1. Initialise the comment notification receipients database table, e.g.
+
+        cd /usr/lib/ckan/default/src/ckanext-ytp-comments # Your PATH may vary
+        
+        paster init_notifications_db -c /etc/ckan/default/development.ini # Use YOUR path and relevant CKAN .ini file
```

Comment:
Can't this be added to the normal InitDBCommand, so we have just one migration command to run for the plugin ?